src/components/BottomBar.tsx

```diff
@@ -0,0 +1,175 @@
+'use client';
+
+import { Box, Typography, Button, Menu, MenuItem } from '@mui/material';
+import Image from 'next/image';
+import Link from 'next/link';
+import { useCallback, useEffect, useState } from 'react';
+
+export default function BottomBar() {
+  const [isOpen, setOpen] = useState<null | HTMLElement>(null);
+  const [currentLanguage, setCurrentLanguage] = useState<string>('Português');
+  const [time, setTime] = useState<string>();
+  const [date, setDate] = useState<string>();
+  const languages = [
+    {
+      name: 'Português',
+      flagUrl: '/portuguese-flag.svg',
+    },
+    {
+      name: 'Español',
+      flagUrl: '/spanish-flag.svg',
+    },
+    {
+      name: 'English',
+      flagUrl: '/english-flag.svg',
+    },
+  ];
+
+  const handleOpen = useCallback(
+    (event: React.MouseEvent<HTMLButtonElement>) => {
+      setOpen(event.currentTarget);
+    },
+    []
+  );
+
+  const handleClose = useCallback(() => {
+    setOpen(null);
+  }, []);
+
+  const formatDate = () => {
+    const currentDate = new Date();
+
+    const options = {
+      weekday: 'long',
+      day: 'numeric',
+      month: 'short',
+    } as const;
+
+    const formattedDate = new Intl.DateTimeFormat('pt-BR', options)
+      .format(currentDate)
+      .replace('-feira', '');
+
+    setDate(formattedDate.charAt(0).toUpperCase() + formattedDate.slice(1));
+  };
+
+  useEffect(() => {
+    setInterval(() => {
+      const currentDate = new Date();
+      const hour = currentDate.getHours();
+      const minute = currentDate.getMinutes();
+      const currentTime = `${hour}:${minute}`;
+
+      setTime(currentTime);
+    }, 1000);
+
+    formatDate();
+  }, []);
```

Comment:
There is an error when any clock value starts with zero. Also, I find it interesting to set `currentDate` outside `setInterval` and share it with `formatDate` function.